apps/jobs/views.py

```diff
@@ -778,9 +778,10 @@ def update_submission(request, challenge_pk):
         if successful_submission:
             try:
                 results = json.loads(submission_result)
-            except ValueError:
+            except (ValueError, TypeError) as exc:
                 response_data = {
-                    "error": "`result` key contains invalid data. Please try again with correct format!"
+                    "message": "`result` key contains invalid data. Please try again with correct format.",
+                    "error": str(exc),
```

Comment:
Why are we sending two keys? I thought we planned on keeping one key in the response_data dict.

If success, then key will be `message` and if there is error, we will have the key `error`